### ISSUE TITLE
Slider defaults

### ISSUE THREAD

**Issue by rwbaer** [makehumancommunity/community-plugins-massproduce#1]
The default behavior for right clicking on a slider in mh is to restore the slider to its default.  In mass produce, a right acts the same as a left click, and it sets the slider to the position clicked.

**Comment by joepal1976** [comment]
Huh. Fascinating. I never knew the right click behavior in the main GUI until now, I guess I never actually right-clicked a slider...

**Comment by rwbaer** [comment]
Yes.  It's great for non-artists like me who have no intuition what a slider will do until they move it.  Inevitably it too late then.  :-)
